### ISSUE TITLE
Settings tabs are awkward

### ISSUE THREAD

**Issue by wrjlewis** [vector-im/element-call#1553]
### Steps to reproduce

On either your mobile or desktop join a call on https://call.element.dev/

Open the settings dialog box

#### What did you expect?
Visually compelling and easily accessible menu options in the setting dialog

#### What happened instead?

**On Desktop**
![Screenshot 2023-09-20 at 13 50 23](https://github.com/vector-im/element-call/assets/1543626/c49c2391-8330-4bd9-a202-236ea0781c87)

There is a vertical scroll bar which hides the remaining options. 
* Adds some amount of inconvenience to navigate to the remaining options that are out of view. 
* Not easy to see which options are hidden out of view
* The horizontal scroll bar isn't very visually appealing / in-line with the gorgeous design everywhere else in the app

**On Mobile through the EX app or on Safari**
![IMG_8961](https://github.com/vector-im/element-call/assets/1543626/fe14ba56-bf05-4931-bd7f-d4c8d3200732)

There is no indication that is a scroll bar and therefore options are hidden from users, in the menu of the settings. 

It is also quite difficult to accurately swipe left and right to move the menu items, even if you do know that more options are there. 

### Operating system

Mac/Safari. iOS/Safari. iOS/ElementX

### URL for webapp

call.element.dev

### Will you send logs?

No

**Comment by nadonomy** [comment]
@fkwp @robintown resources depending - I think this is worth actioning in the coming weeks. If you agree, we could schedule some time to figure out the right solution together. Initial thoughts:

- Should we iterate on the tab implementation to strengthen it - we're using similar tabs in MAS and so on, so this could be fruitful
- Should we collapse into less tabs when viewing responsively

**Comment by axelsimon** [comment]
Thanks for opening this @wrjlewis!

Agree with all of it, just to add: if you enable dev tools, it gets worse, as it adds another option.

This is what it looks like on Firefox 117.01 of Fedora 38:

![Screenshot from 2023-09-20 15-08-34](https://github.com/vector-im/element-call/assets/1172256/7408f1c3-61e1-450c-a915-e6ffd95e1b8b)

**Comment by robintown** [comment]
@nadonomy Agreed, this part of the UI is definitely one of the top candidates for updating next. For what it's worth, this style of tabbing isn't anything new: this is how settings have always looked on mobile prior to the Compound redesign, which is why I felt OK leaving it untouched for the first implementation pass.

Depending on my availability, I'll either jump straight to implementing the new settings designs as a whole, or start with the smaller step of swapping out the tabs component for the Compound one pictured here. From my understanding, this component is the same one you're referring to, and already has better affordances for when the list overflows and becomes scrollable?

![Screenshot from 2023-09-20 10-51-59](https://github.com/vector-im/element-call/assets/48614497/803d5399-d3e1-4022-9ff2-de1a7bd35aa9)

**Comment by nadonomy** [comment]
> Depending on my availability, I'll either jump straight to implementing the new settings designs as a whole, or start with the smaller step of swapping out the tabs component for the Compound one pictured here. From my understanding, this component is the same one you're referring to, and already has better affordances for when the list overflows and becomes scrollable?

I don't think we've got as far as implementing any mobile friendly overflows anywhere yet. We have designed some [here](https://www.figma.com/file/wqgXVb7RVaIybfRsXdWEiL/Trust-%26-Decoration?type=design&node-id=837%3A53617&mode=design&t=0W9huioJrTabHvPX-1), but we could do with pairing (design/engineering) on them as we implement.

**Comment by nadonomy** [comment]
> I don't think we've got as far as implementing any mobile friendly overflows anywhere yet. 

On this - @germain-gg @sandhose if you could correct me if I'm wrong that'd be great!

Relatedly, @dav-is I remember you were experimenting with some MUI components for ESS - which I think for licensing we wouldn't want to include in Compound Web/all projects. But, have you done any R&D here we could build on?

**Comment by dav-is** [comment]
This is how we handle the mobile navigation in the Admin Console:

## Desktop
![Screenshot from 2023-09-20 11-16-36](https://github.com/vector-im/element-call/assets/11590024/41e466d3-cd34-40ca-9316-61bfc71055e9)

## Mobile
![Screenshot from 2023-09-20 11-16-47](https://github.com/vector-im/element-call/assets/11590024/fe782931-a6b4-46d5-8f7b-99827a120e98)

## Animations

(In case it isn't obvious, when I am not pressing the `>` button, I am scrolling using my horizontal scroll wheel)

[Screencast from 2023-09-20 11-27-17.webm](https://github.com/vector-im/element-call/assets/11590024/4afd4f76-8a26-4f97-806e-de134be04b69)

> which I think for licensing we wouldn't want to include in Compound Web/all projects

It's MIT-licensed, so I'm not sure what you mean. I would say we probably shouldn't use MUI in compound due to added complexity.

**Comment by nadonomy** [comment]
> It's MIT-licensed, so I'm not sure what you mean. I would say we probably shouldn't use MUI in compound due to added complexity.

Ah great. I remembered there was _some_ gotcha. Thanks for clarifying.

**Comment by Hywan** [comment]
2 glitches I saw on Firefox/macOS:

* The scrollbar is showing off like a star and takes all the place

![Screenshot 2023-09-21 at 10 02 01](https://github.com/vector-im/element-call/assets/946104/d5a135d0-2384-44d9-8a54-8dfe219328ef)

* The focus on the list dropdown is clipped

![Screenshot 2023-09-21 at 10 02 08](https://github.com/vector-im/element-call/assets/946104/927c85b0-7670-4b74-b94d-ed2660d8c737)
